helm-charts/seldon-core-operator/templates/webhook.yaml

```diff
@@ -34,9 +20,7 @@ webhooks:
   - v1
   - v1beta1
   clientConfig:
-{{- if not .Values.certManager.enabled }}
```

Comment:
It seems this was accidentally removed, may be best to just revert the webhook.yaml diff

Reply:
This is automatically created by the python script so would be a bug if it was removed?

Reply:
Yeah it definitely would be a bug in the python helm generators - we could fix it as a separate pr or if it's a quick fix could just be updated here

Reply:
I reverted for now and added an issue #4217

Reply:
https://github.com/SeldonIO/seldon-core/pull/4218/files